Samples/Client/WebClient/index.html

```diff
@@ -129,11 +213,96 @@
         console.log("setLocalDescription failed");
     }
 
-    function aic_success_cb() {
+    function navOnMouseDown(event)
+    {
+        isMouseDown = true;
+
+        mouseDownX = event.clientX;
+        mouseDownY = event.clientY;
+
+        downPitch = navPitch;
+        downHeading = navHeading;
+        downLocation[0] = navLocation[0];
+        downLocation[1] = navLocation[1];
+        downLocation[2] = navLocation[2];
+
+        preventDefault();
+    }
+
+    function navOnMouseUp()
+    {
+        isMouseDown = false;
+
+        preventDefault();
+    }
+
+    function navOnMouseMove(event)
+    {
+        if (isMouseDown)
+        {
+            var dx = event.clientX - mouseDownX;
+            var dy = event.clientY - mouseDownY;
+            
+            if (event.ctrlKey == 0)
+            {
+                // calc delta pitch & heading change
+                var dpitch = 0.005 * dy;
+                var dheading = 0.005 * dx;
+
+                navHeading = downHeading - dheading;
+                navPitch = downPitch + dpitch;
+
+                // calc delta rotation matrix
+                var locTransform =  matMultiply(matRotateY(navHeading), matRotateZ(navPitch));
+                navTransform = matMultiply(matTranslate(navLocation), locTransform);
+                
+                // send transform message to server
+                sendTransform();
+            }
+            else
+            {
+                 // calculate distance to travel
+                var dist = -dy;
+
+                // update the navigation origin
+                navLocation[0] = downLocation[0] + dist * navTransform[0];
+                navLocation[1] = downLocation[1] + dist * navTransform[1];
+                navLocation[2] = downLocation[2] + dist * navTransform[2];
+                
+                navTransform[12] = navLocation[0];
+                navTransform[13] = navLocation[1];
+                navTransform[14] = navLocation[2];
+
+                // send transform message to server
+                sendTransform();
+            }
+
+            preventDefault();
```

Comment:
do you mean `event.preventDefault()` ?

Reply:
yes, this should be event.preventDefault()